src/unstract/adapters/llm/azure_open_ai/src/azure_open_ai.py

```diff
@@ -51,8 +52,8 @@ def get_json_schema() -> str:
 
     def get_llm_instance(self) -> Optional[LLM]:
         try:
-            model: Optional[str] = self.config.get(Constants.MODEL)
-            llm = AzureOpenAI(
+            llm: Optional[LLM] = AzureOpenAI(
+                model=self.config.get(Constants.MODEL, Constants.DEFAULT_MODEL),
```

Comment:
This is because, AzureOpenAI defaults the model to gpt-35-turbo(DEFAULT_MODEL) if we do not set it. Hence I added the same as default. Otherwise, mypy will complain for type checking saying that model cannot be Optional[str] as the return type of self.config.get() is Optional[..]